ipn/ipnlocal/local.go

```diff
@@ -91,6 +93,14 @@ import (
 	"tailscale.com/wgengine/wgcfg/nmcfg"
 )
 
+var lazyInTest lazy.SyncValue[bool]
+
+func inTest() bool {
+	return lazyInTest.Get(func() bool {
+		return flag.Lookup("test.v") != nil
+	})
+}
```

Comment:
sure, but this should live elsewhere like we have in corp. Then corp can use the oss copy too.

Reply:
yeah, we have a few copies of this in oss. Any package preferences?

Reply:
We have `util/cibuild` for just `func On() bool`.

Something beside that? `util/testenv`? `testenv.InTest()` ?

Reply:
Sent #8815